watch/index.js

```diff
@@ -0,0 +1,42 @@
+let date = new Date();
+const days = ["Sun", "Mon", "Tue", "Wed", "Thu", "Fri", "Sat"];
```

Comment:
constant variables (when it's a static constant value - we name with capital letters)

DAY_NAMES